site2/website/siteConfig.js

```diff
@@ -95,6 +98,7 @@ const siteConfig = {
     {page: 'download', label: 'Download'},
     {doc: 'client-libraries', label: 'Clients'},
     {page: 'admin-rest-api', label: 'REST API'},
+    {page: 'functions-rest-api', label: 'Functions REST API'},
```

Comment:
This will make the head link bar a bit long. 

We could do like for the "Community" and "Apache" that use a dropdown list with multiple links.